bindata/network/iptables-alerter/002-script.yaml

```
@@ -37,36 +24,31 @@ data:
         date
         # Iterate over local pods
         for id in $(crictl pods -q); do
-            # Check that it's a pod-network pod
-            netns=$(crictl inspectp "${id}" | jq -r .status.linux.namespaces.options.network)
+            # Inspect the pod
+            read pod_namespace pod_name pod_uid netns netns_path <<<$(
+                {{- /* bindata/network/iptables-alerter/002-script.yaml is processed
+                     * as a go template (eg, to fill in {{.ReleaseVersion}} above), but
+                     * it also needs to contain a go template string (to pass to crictl
+                     * below). We do this by putting the crictl argument as a string
+                     * literal inside an argument of the outer template:
+                     *     ... --template {{`'BLAH BLAH BLAH'`}} ...
+                     * becomes
+                     *     ... --template 'BLAH BLAH BLAH' ...
+                     * in the actual ConfigMap.
+                     */
+                -}}
```

The template doesn't work:
```
template: ../../bindata/network/iptables-alerter/002-script.yaml:29: comment ends before closing delimiter"
```
This should work:
```suggestion
                {{- /* 
                     * bindata/network/iptables-alerter/002-script.yaml is processed
                     * as a go template (eg, to fill in {{.ReleaseVersion}} above), but
                     * it also needs to contain a go template string (to pass to crictl
                     * below). We do this by putting the crictl argument as a string
                     * literal inside an argument of the outer template:
                     *     ... --template {{`'BLAH BLAH BLAH'`}} ...
                     * becomes
                     *     ... --template 'BLAH BLAH BLAH' ...
                     * in the actual ConfigMap.
                */ -}}
```